golem-worker-executor-base/tests/api.rs

```diff
@@ -1745,26 +1765,25 @@ async fn long_running_poll_loop_works_as_expected(
 
     let response = Arc::new(Mutex::new("initial".to_string()));
     let response_clone = response.clone();
-    let host_http_port = context.host_http_port();
-
-    let http_server = tokio::spawn(async move {
-        let route = Router::new().route(
-            "/poll",
-            get(move || async move {
-                let body = response_clone.lock().unwrap();
-                body.clone()
-            }),
-        );
-
-        let listener = tokio::net::TcpListener::bind(
-            format!("0.0.0.0:{}", host_http_port)
-                .parse::<SocketAddr>()
-                .unwrap(),
-        )
-        .await
-        .unwrap();
-        axum::serve(listener, route).await.unwrap();
-    });
+
+    let listener = tokio::net::TcpListener::bind("0.0.0.0:0").await.unwrap();
```

Comment:
Changed all of these to work without fixed ports. Given the number of tests we have, we had to keep quite a large range of ports free